src/components/Login/Login.tsx

```diff
@@ -0,0 +1,104 @@
+import "./Login.css";
+import React, { useState } from "react";
+
+// username: "mor_2314",
+// password: "83r5^_",
+const URL = "https://fakestoreapi.com/auth/login";
+const userData = ["User123", "abcd1234"]
+
+interface LoginProps {
+  loginFlagHandler: (args: boolean) => void;
+}
+
+const Login = ({ loginFlagHandler }: LoginProps) => {
+  const [user, setUser] = useState<string>("");
```

Comment:
Please refactor the form handling with useReduce instead of using multiple useState.

Reply:
addressed the comment.